docs/how-tos/production_configuration.rst

```diff
@@ -0,0 +1,245 @@
+- (Kafka) Can potentially `integrate directly`_ with ClickHouse removing the need for both a separate event consumer and Ralph.
+
+Cons:
+
+- Running consumers is not currently handled automatically by Aspects
+- Many parts are new and may not have extensive production testing
+
+Recommended for:
+
+Large-to-very-large instances, adventurous site operators, installations that already have Kafka or advanced use cases that can benefit from a multi-consumer architecture.
+
+
+Setting up the xAPI Pipeline
+############################
+
+**Celery**
+
+When in doubt, the simplest place to start with a production configuration is Celery tasks with batching set to 100 events or 5 seconds. You will want to add at least one additional lms-worker to handle the additional load of xAPI events and the event sink pipeline. You will also probably want to add at least one additional cms-worker to handle the new work of the course publishing event sink.
+
+**Vector**
+
+Generally the Aspects created Vector configuration should work in most cases. In Kubernetes environments you will need to make sure that a Vector pod is attached to each LMS/CMS worker.
+
+**Event bus**
+
+Similar to Celery, you should start with at least 2 event bus consumers and configure batching to 100 events or 5 seconds to start with. If you find that the event queue size is growing (see "Monitoring", below), you can add more event bus consumers and/or increase the batch size. We have tested with batch sizes up to 1000 without issue.
+
+
+Choosing ClickHouse Hosting
+###########################
+
+By default Aspects deploys a single ClickHouse Docker container as part of the Tutor install. This is not the preferred way to run a production environment! In most cases, if you can afford it, ClickHouse Cloud is the easiest and most performant way to run the service, and removes the burden of dealing with scaling, security, upgrades, backups, and other potentially difficult database management issues. Axim has been using ClickHouse Cloud for load testing and is designed to work with it.
+
+Altinity Cloud is another hosting service that Aspects has tested with in the past, but may require more hands-on integration as they use a different clustering approach than ClickHouse Cloud.
+
+Another option if you are running in Kubernetes is to use the `clickhouse-operator`_ to deploy and manage a more fault tolerant ClickHouse cluster. Aspects support for ClickHouse clusters is currently experimental, and may not support all cluster configurations without modification.
+
+
+Setting up ClickHouse
+#####################
+
+Tutor local and k8s environments should work out of the box. See Remote ClickHouse <remote-clickhouse> and ClickHouse Cluster <clickhouse-cluster> for more information on setting up hosted services.
+
+.. note::
+
+    Don't forget the usual checklist items! Make sure the server is secured, only accessible from places it needs to be, and backed up!
```

Comment:
We don't support ClickHouse backups, we assume all data can be recovered. Leave a warning for operators to configure a tracking log backup solution according to their needs

Reply:
We don't natively support backups, but people are able to backup / restore using native ClickHouse tools and get it automatically from ClickHouse Cloud